tools/query_sloop.c

```diff
@@ -0,0 +1,12 @@
+// Query if player has sloop shadow on them
+// Gon
+// 2017 - 04
+
+// Declared Functions
+int query_sloop();
+
+// Find if target has sloop shadow and return 1
+int query_sloop( object victim ) {
+    return sizeof( filter( shadow_list( victim ), (: if(blueprint($1) == FINDO(
```

Comment:
So yeah, this one is tricky. Basically I was working with Coz on different ways to go about identifying a specific shadow. My code here is a version of something he gave to me to run using CC. You're absolutely correct in that a lot of this is over my head. He briefly talked to me about closures and I sort of get it, but the only thing I've actually done with this code is load it and not get any errors. I haven't yet gotten around to actually calling it yet. I plan on doing that directly from isolationchamber.c.

Yeah, and Coz and I both saw that this was the case if the blueprint got dested. Good to know about program_name()! I'll see if I can't redo this tonight.